_overviews/scala3-book/scala-for-python-devs.md

```diff
@@ -0,0 +1,1041 @@
+        <br>&nbsp; "c" -&gt; 3
+        <br>)</code>
+      </td>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row">Access elements</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>my_dict['a']&nbsp;&nbsp; # 1</code></td>
+      <td valign="top"><code>myMap("a")&nbsp;&nbsp; // 1</code></td>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row">Dictionary/Map with a <code>for</code> loop</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>for key, value in my_dict.items():
+        <br>&nbsp; print(key)
+        <br>&nbsp; print(value)</code>
+      </td>
+      <td valign="top"><code>for
+        <br>&nbsp; (key,value) &lt;- myMap
+        <br>do
+        <br>&nbsp; println(key)
+        <br>&nbsp; println(value)</code>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+Scala has other specialized `Map` classes for different needs.  
+
+
+### Sets
+
+The Python set is like the mutable Scala `Set` class.
+
+<table cellspacing="1" cellpadding="2" border="1">
+  <tbody>
+
+    <tr>
+      <th valign="top">Python</th>
+      <th valign="top">Scala</th>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row">Creation</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>set = {"a", "b", "c"}</code></td>
+      <td valign="top"><code>val set = Set(1,2,3)</code></td>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row">Duplicate elements</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>set = {1,2,1}
+        <br># set: {1,2}</code></td>
+      <td valign="top"><code>val set = Set(1,2,1)
+        <br>// set: Set(1,2)</code>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+Scala has other specialized `Set` classes for different needs.
+
+
+### Tuples 
+
+The Python and Scala tuples are also similar:
+
+<table cellspacing="1" cellpadding="2" border="1">
+  <tbody>
+    <tr>
+      <th valign="top">Python</th>
+      <th valign="top">Scala</th>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row">Creation</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>t = (11, 11.0, "Eleven")</code></td>
+      <td valign="top"><code>val t = (11, 11.0, "Eleven")</code></td>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row">Access elements</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>t[0]&nbsp;&nbsp; # 11
+        <br>t[1]&nbsp;&nbsp; # 11.0</code>
+      </td>
+      <td valign="top"><code>t(0)&nbsp;&nbsp; // 11
+        <br>t(1)&nbsp;&nbsp; // 11.0</code>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+
+## Methods on collections classes
+
+Python and Scala have several of the same common functional methods available to them:
+
+- `map`
+- `filter`
+- `reduce`  
+
+If you’re used to using these methods with lambda expressions in Python, you’ll see that Scala has a similar approach with methods on its collections classes. To demonstrate this functionality, here are two sample lists:  
+
+```scala
+list = (1,2,3)           // python
```

Comment:
```suggestion
numbers = (1,2,3)           // python
```
list is an already existing method, so I would avoid using it here.

Reply:
Thanks! If we change this, we also must not forget the references to `list` below.